CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
@pmolodo Was the `cmake` bump intentional?

Reply:
Nope: [undo accidental cmake version bump](https://github.com/PixarAnimationStudios/OpenUSD/pull/3085/commits/95e2cad44c9c329a221cd7c89f62a5d4de1a12d6)